tests/localnet/test_localnet_start.py

```diff
@@ -12,7 +18,42 @@
 from tests.utils.proc_mock import ProcMock
 
 
-@pytest.mark.usefixtures("proc_mock", "health_success")
+@pytest.fixture()
+def _localnet_out_of_date(proc_mock: ProcMock, httpx_mock: HTTPXMock) -> None:
+    arg = '{{index (split (index .RepoDigests 0) "@") 1}}'
+    proc_mock.set_output(
+        ["docker", "image", "inspect", ALGORAND_IMAGE, "--format", arg],
+        ["[algorand/algod@sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb\n]"],
```

Comment:
Should be just `sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb\n` without square brackets due to `{{index (split (index .RepoDigests 0) "@") 1}}`

Reply:
These are still in place? Am i missing something? I think you pushed a commit to remove them from the test but unsure if these are still correct, the output should not contain [{title}/{image title}@sha256:{hash}] = we parse it out via go templating in format param.